docs/gettingstarted/index.rst

```diff
@@ -173,7 +173,9 @@ Getting started with Flocker
             you@laptop:~$ git clone \
               https://github.com/clusterhq/vagrant-flocker && \
               cd vagrant-flocker && \
-              vagrant up
+              vagrant up && \
+              [ -e "${SSH_AUTH_SOCK}" ] || eval $(ssh-agent) && \
+              ssh-add ~/.vagrant.d/insecure_private_key
```

Comment:
The docs don't tell you to launch multiple shells.  It's certainly true that if you do, this isn't going to help you.  We could explicitly tell people to run all the commands in one shell.  I wonder what prompted the idea that you should use multiple shells for the guide, though?

Since this is what's actually deployed now, I don't think this should block the merge (but that doesn't mean we shouldn't do some follow-up work to make this better).